### PR TITLE
Beeswax Install Script 

### PR DESCRIPTION
**Pull request by kabilans**
Request to push our honeypot's (Beeswax) deploy script upstream. We added MHN support and would like the deploy script upstream. Please let us know if you require anything else. @iankronquist

### CODE REVIEW

**Comment by jt6211** [comment]
Can you include your beeswax deploy script in this PR?

**Comment by iankronquist** [comment]
@jt6211 Go ahead and close this.  I will reopen it with all of the required files later today.

**Comment by iankronquist** [comment]
(I am working on the same team with Sibi for a senior project)

**Comment by kabilans** [comment]
Committed the install scripts and added the deploy scripts. Sorry about the mixup!

**Comment by jt6211** [comment]
This looks good, but in order for the data to show up in MHN you also need to do the rest of this:  https://github.com/threatstream/mhn/wiki/Howto%3A-Add-Support-for-New-Sensors-to-the-MHN

**Comment by pixel5919** [comment]
I just queried a fresh server of @kabilans MHN in MongoDB with `db.auth_key.find({"identifier":"mnemosyne"})` & `db.auth_key.find({"identifier":"geoloc"})` and the honeypot was present

**Comment by jt6211** [comment]
In order for the events to show up in MHN, you need to complete the other steps.  Specifically:

* Without extending mnemosyne, there will be no attacks from this honeypot in MHN's attacks page
* Without extending the geoloc script, none of the beeswax events will get rendered on Honeymap

**Comment by jt6211** [comment]
LGTM

File: scripts/deploy_beeswax.sh [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+	chown -R beeswax $INSTALL_PATH
+}
+
+make_configs() {
+        echo "!!!!!!!!THIS IS IT BOYS!!!!!!!!"
+
+        wget $SERVER_URL/static/registration.txt -O registration.sh
+        chmod 755 registration.sh
+        # Note: this will export the HPF_* variables
+        ./registration.sh $SERVER_URL $DEPLOY_KEY "beeswax"
```
Comment (pixel5919) [inline scripts/deploy_beeswax.sh]: Add the dot back, I thought it was a typo, didn't know that it meant source, Ubuntu 14.04 wasn't working correctly

**Comment by d1str0** [comment]
@kabilans is Beeswax still supported? If we were to merge this, would you continue to support Beeswax?

**Comment by iankronquist** [comment]
@d1str0 Hi I designed Beeswax. It's no longer maintained, and I would recommend against merging this at this point.

**Pull request closed without merging**